app/build.gradle.kts

```diff
@@ -101,6 +101,7 @@ dependencies {
         libs.appcompat,
         libs.appauth,
         libs.kotlinx.serialization.json,
+        libs.network,
```

Comment:
I'm quite nervous about importing core modules into other core modules, it feels like a violation of dependency inversion principle. Let's have a chat on Monday 🙂

Reply:
Network module has been removed along with the build logic to include that project.